Tools/gz_sim.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+# run multiple instances of the 'px4' binary, with the gazebo SITL simulation
+# It assumes px4 is already built, with 'make px4_sitl_default gazebo'
+
+# The simulator is expected to send to TCP port 4560+i for i in [0, N-1]
+# For example gazebo can be run like this:
+#./Tools/gazebo_sitl_multiple_run.sh -n 10 -m iris
```

Comment:
This seems like a copy paste coming over from another script

Reply:
I think if you were to look at it more closely you would see that it's a superset of all potential uses/runs for gazebo, hence why parts might look familiar. However, unlike the others, it allows for HITL and SITL, gen from json for both worlds and models (SITL and HITL), setting of simulation threads with smart auto-limits, and much more. Currently there is nothing that provides even a fraction of this functionality.

Reply:
@bperseghetti I didn't mean that you copy pasted the whole script... The comment says the script can be run with `./Tools/gazebo_sitl_multiple_run.sh -n 10 -m iris`

Reply:
Unless you are specifically only talking about the comments, in which case they could be expanded more, but I think it would be best in the form of a dev guide update, the comment section of this otherwise would be very significant. Since this will be replacing the multiple_sitl_gazebo.sh I wanted to let people know that the same functionality is still preserved.

Reply:
@bperseghetti Yes, I was specificly talking about the comments - The comments are not talking about the new script, but explaining how to run the old script.

Reply:
Ok, thanks for clarification, I have updated the comments.

Reply:
Thanks